test/unit/config/ConfigBuilderTest.js

```diff
@@ -152,7 +153,7 @@ describe('ConfigBuilderTest', function () {
         expect(properties['hazelcast.invalidation.reconciliation.interval.seconds']).equal(50);
         expect(properties['hazelcast.invalidation.max.tolerated.miss.count']).to.equal(15);
         expect(properties['hazelcast.invalidation.min.reconciliation.interval.seconds']).to.equal(60);
-        expect(properties['hazelcast.logging.level']).to.be.equal('OFF');
+        expect(properties['hazelcast.logging.level']).to.be.equal(LogLevel.OFF);
```

Comment:
Can we add some tests for the property validation and make sure that it works as expected?